README.md

```diff
@@ -1000,7 +1005,10 @@ Table of Contents
   * [Zeplin](https://zeplin.io/) — Designer and developer collaboration platform. Show designs, assets and styleguides. Free for 1 project.
   * [Pixelixe](https://pixelixe.com/) — Create and edit engaging and unique graphics and images online.
   * [Responsively App](https://responsively.app) - A free dev-tool for faster and precise responsive web application development.
-  * [SceneLab](https://scenelab.io) - Online mockup graphics editor with an ever-expanding collection of free design templates
+  * [SceneLab](https://scenelab.io) - Online mockup graphics editor with an ever-expanding collection of free design templates.
+  * [Siter.io](https://siter.io/) - Design in the cloud and create new websites seamlessly, without any coding skills.
```

Comment:
this seems nice but I cant find information about what "start for free" means, did you find some link that describes the free plan?